sklearn/decomposition/tests/test_fastica.py

```diff
@@ -174,6 +179,10 @@ def test_fastica_convergence_fail():
         ica.fit(m.T)
 
 
+# FIXME remove filter in 1.3
+@pytest.mark.filterwarnings(
+    "ignore:From version 1.3 whiten='unit-variance' will be used by default."
+)
```

Comment:
I would rather pass `whiten='unit-variance'` explicitly to the fast-ica call to make sure this test passes with the future behavior and avoid bad surprises later.

Reply:
If not, we can either adapt the tests or fix `whiten='arbitrary-variance'` right away.